core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -187,6 +187,11 @@ class DAGScheduler(
   /** If enabled, FetchFailed will not cause stage retry, in order to surface the problem. */
   private val disallowStageRetryForTest = sc.getConf.getBoolean("spark.test.noStageRetry", false)
 
+  /** Number of consecutive fetch failures allowed before a stage is aborted */
```

Comment:
Can you change this and the comment on line 1738 to the comment Imran suggested for the docs?